infrastructure-scripts/Create-SqlDbServiceAccount.ps1

```diff
@@ -0,0 +1,139 @@
+<#
+.SYNOPSIS
+Create A Sql Database Service Account Automation
+
+.DESCRIPTION
+Create A Sql Database Service Account Automation
+
+.PARAMETER ServerNamr
+Name of the SQL Server 
+
+.PARAMETER AzureFirewallName
+Name of the tempoary Sql Server Firewall rule created 
+
+.PARAMETER SqlUserName
+Sql Server Master User Name 
+
+.PARAMETER SqlPassword
+Sql Server Master Password.
+
+.PARAMETER SqlServiceAccountName
+The name of the service account to be created.
+
+.Parameter SQLResourceGroup
+The Resource group of the SQL Server
+
+.PARAMETER Enviroment
+The Enviroment of the New Service account,
+
+.PARAMETER KeyVaultName
+The name of the Keyvault for the Enviroment
+
+
+.EXAMPLE
+
+.\Create-SqlDbServiceAccount.ps1  -ServerName ServerName `
+                                   -DataBaseName DataBaseName `
+                                   -SqlUserName SqlUserName `
+                                   -SqlPassword SqlPassword `
+                                   -SqlServiceAccountName SqlServiceAccountName `
+                                   -SQLResourceGroup SQLResourceGroup
+                                   -Enviroment Enviroment
+                                   -KeyVaultName KeyVaultName
+
+#>
+
+[CmdletBinding(DefaultParameterSetName = 'None')]
+param
+(
+  [String] [Parameter(Mandatory = $true)] $ServerName,
+  [String] $AzureFirewallName = "AzureWebAppFirewall",
+  [String] [Parameter(Mandatory = $true)] $DataBaseName,
+  [String] [Parameter(Mandatory = $true)] $SqlUserName,
+  [SecureString] [Parameter(Mandatory = $true)] $SqlPassword,
+  [String] [Parameter(Mandatory = $true)] $SqlServiceAccountName,
+  [String] [Parameter(Mandatory = $true)] $SQLResourceGroup,
+  [String] [Parameter(Mandatory = $true)] $Enviroment,
+  [String] [Parameter(Mandatory = $true)] $KeyVaultName
+)
```

Comment:
Perhaps format the parameters like the other scripts we have for consistency:
https://github.com/SkillsFundingAgency/das-platform-automation/blob/add_infrastructure_scripts/infrastructure-scripts/New-ServiceBusConnectionString.ps1#L31-L43

Reply:
Done